tests/compas/geometry/test_tranformations/test_helpers.py

```diff
@@ -0,0 +1,63 @@
+import pytest
```

Comment:
the name of this folder (`test_tranformations`) seems to be wrong => `test_transformations`

Reply:
true for all other test, but will not repeat it...

Reply:
my bad, thats embarrassing...